src/app/ansyn/ansyn/ansyn.component.less

```diff
@@ -13,13 +13,15 @@
 			flex-direction: column;
 			overflow: hidden;
 			position: relative;
-			.maps-container {
+
+.maps-container {
```

Comment:
If it finds a rule that is adjacent to another one, it creates a new line, but also trims that line so it ignores indentation. (this is consistent throughout all of the code)